pkg/compute/sensors/completed_jobs.go

```diff
@@ -0,0 +1,29 @@
+package sensors
+
+import (
+	"context"
+
+	"github.com/filecoin-project/bacalhau/pkg/compute/store"
+	"github.com/filecoin-project/bacalhau/pkg/model"
+)
+
+type CompletedJobs struct {
```

Comment:
Naming: this should probably be `CompletedJobProvider` to match the naming scheme of things being "debug info providers".